TwilioVerifySDK/TwilioVerify/Sources/Domain/Challenge/PushChallengeProcessor.swift

```diff
@@ -29,32 +29,58 @@ class PushChallengeProcessor {
   private let challengeProvider: ChallengeProvider
   private let jwtGenerator: JwtGeneratorProtocol
   
-  init(challengeProvider: ChallengeProvider, jwtGenerator: JwtGeneratorProtocol = JwtGenerator()) {
+  init(
+    challengeProvider: ChallengeProvider,
+    jwtGenerator: JwtGeneratorProtocol
+  ) {
     self.challengeProvider = challengeProvider
     self.jwtGenerator = jwtGenerator
   }
 }
 
 extension PushChallengeProcessor: PushChallengeProcessorProtocol {
-  func getChallenge(withSid sid: String, withFactor factor: PushFactor, success: @escaping ChallengeSuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func getChallenge(
+    withSid sid: String,
+    withFactor factor: PushFactor,
+    success: @escaping ChallengeSuccessBlock,
+    failure: @escaping TwilioVerifyErrorBlock
+  ) {
     Logger.shared.log(withLevel: .info, message: "Getting challenge \(sid) with factor \(factor.sid)")
     challengeProvider.get(withSid: sid, withFactor: factor, success: success, failure: { error in
       failure(TwilioVerifyError.inputError(error: error as NSError))
     })
   }
   
-  func updateChallenge(withSid sid: String, withFactor factor: PushFactor, status: ChallengeStatus, success: @escaping EmptySuccessBlock, failure: @escaping TwilioVerifyErrorBlock) {
+  func updateChallenge(
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/0307d478e851ddee7e698ae03678695d1c0f326c/TwilioVerifySDK/TwilioVerify/Sources/Domain/Challenge/PushChallengeProcessor.swift#L54"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 60 lines<br />
<code>function_body_length</code> <code>PushChallengeProcessor.swift:54</code></td>
    </tr>
  </tbody>
</table>